AdblockPlusSafari/AppDelegate.m

```diff
@@ -20,14 +20,23 @@
 #import "Appearence.h"
 #import "RootController.h"
 
+const NSTimeInterval FilterlistsUpdatePeriod = 3600*24;
```

Comment:
We should go with 4 days here, for now. EasyList itself has a mechanism for reducing the download interval in case there is just too much traffic - here we don't have that.